app/controllers/converts_controller.rb

```diff
@@ -28,6 +28,9 @@ def create
       filename: file_name,
       layout: false
     )
+  rescue ActionController::ParameterMissing
+    flash.now[:alert] = "Missing 'file' or 'format' form option."
```

Comment:
Ah I see, so then `flash.now` would be used when we `render` and `flash` would be used when we `redirect` typically. Cool, thanks for explaining, this makes sense.